runtime/fastly/builtins/fastly.h

```diff
@@ -0,0 +1,65 @@
+#ifndef FASTLY_FASTLY_H
+#define FASTLY_FASTLY_H
+
+#include "../../StarlingMonkey/builtins/web/url.h"
+#include "../host-api/host_api_fastly.h"
+#include "./fetch/request-response.h"
+#include "builtin.h"
+#include "extension-api.h"
+#include "fastly.h"
+#include "host_api.h"
+
+using namespace builtins;
+
+namespace fastly::fastly {
+
+class Env : public BuiltinNoConstructor<Env> {
+private:
+  static bool env_get(JSContext *cx, unsigned argc, JS::Value *vp);
+
+public:
+  static constexpr const char *class_name = "Env";
+
+  static const JSFunctionSpec static_methods[];
+  static const JSPropertySpec static_properties[];
+  static const JSFunctionSpec methods[];
+  static const JSPropertySpec properties[];
+
+  static JSObject *create(JSContext *cx);
+};
+
+const JSErrorFormatString *FastlyGetErrorMessage(void *userRef, unsigned errorNumber);
+
+class Fastly : public BuiltinNoConstructor<Fastly> {
+private:
+  // static bool log(JSContext *cx, unsigned argc, JS::Value *vp);
+
+public:
+  static constexpr const char *class_name = "Fastly";
+
+  static JS::PersistentRooted<JSObject *> env;
+  static JS::PersistentRooted<JSObject *> baseURL;
+  static JS::PersistentRooted<JSString *> defaultBackend;
+  static bool allowDynamicBackends;
+
+  static const JSPropertySpec properties[];
+
+  // static bool createFanoutHandoff(JSContext *cx, unsigned argc, JS::Value *vp);
+  // static bool now(JSContext *cx, unsigned argc, JS::Value *vp);
+  // static bool dump(JSContext *cx, unsigned argc, JS::Value *vp);
+  // static bool enableDebugLogging(JSContext *cx, unsigned argc, JS::Value *vp);
```

Comment:
Could we get some TODOs on the commented-out code to make it obvious that it's part of an in-progress migration?

Reply:
Added TODOs to all code that should be uncommented.